### ISSUE TITLE
Couldn't start screen recording (all permissions are granted) !

### ISSUE THREAD

**Issue by Belveloper** [endmr11/ed_screen_recorder#87]

**Comment by jdevp** [comment]
I ran into the same issue on one of my Android devices, running Android 10, SDK29.  When this happens, the `stopRecord` returns no message other than a message of `fail to release resource`.  Unfortunately, this makes this package useless till this issue is fixed.

**Comment by Belveloper** [comment]
Yes, we hope Flutter team ( or Community  ) make a good screen record flutter package or the author improves this one (it's perfect when it works hhhhhh ).

**Comment by HBiSoft** [comment]
@Belveloper 
There's nothing stopping you from creating one yourself or contributing to this library.

**Comment by Belveloper** [comment]
> @Belveloper 
> There's nothing stopping you from creating one yourself or contributing to this library.

Yes, I'll do my best

**Comment by jdevp** [comment]
I'm writing my own plugin. This plugin has an issue by using HBRecorder library, which works on some phones but not others. HBRecorder uses HIGH QUALITY profile to record video but it doesn't handle the cases when screen sizes are larger than video buffer. It only works for the devices where the screen dimensions can into it the video buffer. Most newer phones have larger video buffer.
